docs/pages/router/basics/real-life-nav.mdx

```diff
@@ -0,0 +1,203 @@
+    ['app/(tabs)/feed/_layout.tsx', 'tab with a stack inside'],
+    ['app/(tabs)/feed/index.tsx'],
+    ['app/(tabs)/feed/[postId].tsx'],
+    ['app/(tabs)/settings.tsx', 'single page tab'],
+  ]}
+/>
+
+In the **app/(tabs)\_layout.tsx** file, return a `Tabs` component:
+
+```tsx _layout.tsx
+import { Tabs } from 'expo-router';
+
+export default function TabLayout() {
+  return (
+    <Tabs>
+      <Tab.Screen name="index" options={{ title: 'Home' }} />
+      <Tab.Screen name="feed" options={{ title: 'Feed' }} />
+      <Tab.Screen name="settings" options={{ title: 'Settings' }} />
+    </Tabs>
+  );
+}
+```
+
+In the **app/(tabs)/feed/\_layout.tsx** file, return a `Stack` component:
+
+```tsx feed/_layout.tsx
+import { Stack } from 'expo-router';
+
+/* @info Setting initialRouteName ensures that direct links deep into the stack still push the index route onto the stack first. */
+export const unstable_settings = {
+  initialRouteName: 'index',
+};
+/* @end */
+
+export default function FeedLayout() {
+  return <Stack />;
+}
+```
+
+Now, within the **app/(tabs)/feed** folder, you can have `Link` components that point to different posts (e.g., `/feed/123`). Those links will push the `feed/[postId]` route onto the stack, leaving the tab navigator visible.
+
+You can also navigate from any other tab to a post in the feed tab with the same URL. Use `withAnchor` in conjunction with `initialRouteName` to ensure that the `feed/index` route is always the first screen in the stack:
+
+```tsx app/(tabs)/feed/index.tsx
+<Link href="/feed/123" withAnchor>
+  Go to post
+</Link>
+```
+
+You can also nest tabs inside of an outer stack navigator. That is often more useful for displaying modals over the tabs.
+
+[Learn more about nested navigators](/router/advanced/nested-navigators/).
+
+## One screen, two tabs: sharing routes
+
+Route groups can be used to share a single screen between two different tabs. Consider a navigation tree where there is a Feed tab and a Search tab, and they both share pages for viewing a user profile:
+
+<FileTree
+  files={[
+    ['app/(tabs)/\_layout.tsx'],
+    ['app/(tabs)/(feed)/index.tsx', 'default route'],
+    ['app/(tabs)/(search)/search.tsx'],
+    ['app/(tabs)/(feed,search)/_layout.tsx', 'layout shared between the two tabs'],
+    ['app/(tabs)/(feed,search)/users/[username].tsx', 'shared user profile page'],
+  ]}
+/>
+
+Each of the tabs is put in a group so we can define a third folder that shares routes between two groups (**app/(tabs)/(feed,search)**). Even with the extra layer, **app/(tabs)/(feed)/index.tsx** is still the nearest index, so it will be the default route.
+
+```tsx (tabs)/_layout.tsx
+import { Tab } from 'expo-router';
+
+export default function TabLayout() {
+  return (
+    <Tabs>
+      <Tab name="(feed)" title="Feed" />
+      <Tab name="search" title="Search" />
+    </Tabs>
+  );
+}
+```
+
+Both the `(feed)` and `(search)` route groups contain stacks, so they can share a single layout:
```

Comment:
Both this and the docs example share a layout file which makes it seem like that's the only way to do it, but actually you could use it to just add shared screens as well. Could se make that distinction here? Like "so they _could also_ share a single layout file"

Reply:
took a stab at this